app_dart/lib/src/model/luci/buildbucket.dart

```diff
@@ -650,3 +662,25 @@ enum Trinary {
   @JsonValue('UNSET')
   unset,
 }
+
+/// A requested dimension. Looks like StringPair, but also has an expiration.
+@JsonSerializable(includeIfNull: false)
+class RequestedDimension extends JsonBody {
```

Comment:
Isn't this basically a map?

Reply:
Kind of, this is defined in https://chromium.googlesource.com/infra/luci/luci-go/+/master/buildbucket/proto/common.proto.

Reply:
How does the string gets translated to the proto value?